Sources/FileManagerExtension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  FileManagerExtension.swift
+//  EZSwiftExtensions
+//
+//  Created by Albert Vila on 16/11/16.
+//  Copyright © 2016 Goktug Yilmaz. All rights reserved.
+//
+
+import Foundation
+
+extension FileManager {
+    
+    /// EZSE: Returns path of documents directory
+    public class func documentsDirectoryPath() -> String {
```

Comment:
These could very well be vars instead of class funcs. 

Also we can squash commits to 1 as it does not convey any extra information.